rentals.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] From the class Rental under the def initialize you are missing the following implementation:

`*The rental is adding itself to the @book achieving the 'belongs-to' association`
`*The rental is adding itself to the @person the 'belongs-to' association`

Reply:
- [ ] Please implement this requirement asked by the previous reviewer. It is done by doing something similar to:
    `@book = book`
    *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    `@person = person`
    *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self